.gitignore

```diff
@@ -183,4 +183,5 @@ install*/
 /sorc/gdas
 /sorc/gdas-utils
 !/bundle/CMakeLists.txt
+!/bundle/fv3-interface.cmake
```

Comment:
This line is added but it is commented out.  Why?

Reply:
It is not commented out.  This file is ignored from the gitignore list.

Reply:
Thank you @aerorahul for the correction.  @CoryMartin-NOAA If `/bundle/fv3-interface.cmake` is ignored from `.gitignore`, why include it?  A similar question applies to `!/bundle/CMakeLists.txt`.

Reply:
It's "not ignore this file" because otherwise every file in the directory /bundle/ is ignored by git

Reply:
OK.  The .gitmodule documentation explains `!`

> An optional prefix "!" which negates the pattern; any matching file excluded by a previous pattern will become included again. It is not possible to re-include a file if a parent directory of that file is excluded. Git doesn’t list excluded directories for performance reasons, so any patterns on contained files have no effect, no matter where they are defined. Put a backslash ("\") in front of the first "!" for patterns that begin with a literal "!", for example, "\!important!.txt".